### ISSUE TITLE
unabled to read source map for http://localhost:4200/runtime.js

### ISSUE THREAD

**Issue by gitppl** [microsoft/vscode#176732]
Hi !
Since yesterday I can't debug anymore and I can't identify the sequence of changes which lead to this issue.
"trace": true set in launch.json gives that error :  (may be related to #167353 / #1336)

**unabled to read source map for http://localhost:4200/runtime.js : Unexpected 503 response from http://localhost:4200/runtime.js.map: connect ECONNREFUSED 127.0.0.1:4200**
(dump file attached)

[vscode-debugadapter-95ebca06.json.gz](https://github.com/microsoft/vscode/files/10939602/vscode-debugadapter-95ebca06.json.gz)

my config :
Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1500.4
@angular-devkit/build-angular   15.0.4
@angular-devkit/core            15.0.4
@angular-devkit/schematics      15.0.4
@schematics/angular             15.0.4
rxjs                            7.5.7
typescript                      4.8.4
Node.js v18.15.0

While running : curl -I http://localhost:4200/vendor.js.map
HTTP/1.1 200 OK
X-Powered-By: Express
Access-Control-Allow-Origin: *
Content-Type: application/json; charset=utf-8
Accept-Ranges: bytes
Content-Length: 4020779
ETag: W/"3d5a2b-XOADmc4FZmXlf23qs4AGEJk9Vsw"
Date: Fri, 10 Mar 2023 06:38:23 GMT
Connection: keep-alive
Keep-Alive: timeout=5

Thanks for your help

**Comment by gitppl** [comment]
Some additional details :

The problem remains the same on a brand new application. App start well but the debug console shows mapping  errors

![screenshot 2023-03-13 071904](https://user-images.githubusercontent.com/30547175/224623062-1892825c-7533-4e94-9713-7fcb95cf4a29.png)

And if I set a breakpoint the application stop working and the breakpoint stay unreachable 

![screenshot 2023-03-13 072751](https://user-images.githubusercontent.com/30547175/224624300-4f76bcce-164a-4be8-ad40-44eea468110d.png)

finally, the debug doc doesn't help me so much...

![screenshot 2023-03-13 073202](https://user-images.githubusercontent.com/30547175/224625261-4f932ec6-fbeb-4bd0-b31c-f570914bb97e.png)

... as sourceMap is well defined in the tsconfig
![screenshot 2023-03-13 074425](https://user-images.githubusercontent.com/30547175/224626854-eda2b801-84ab-40e5-a55a-7c64e57648ef.png)

I reinstalled vscode in doubt, same thing. I really don't know which way to look 😫

**Comment by AndrewStaggPlunk** [comment]
I'm getting the same issue - all of a sudden, 3 days a go, 2 different projects on 2 different machines exhibited the same behaviour.
If you put localhost:4200 into another browser the project runs but it won't debug.
Seems to be only the browser launched by vs code.

As far as I can tell this coincides with 1.76.1 being installed.

Is there a way to download older versions?

**Comment by gitppl** [comment]
> I'm getting the same issue - all of a sudden, 3 days a go, 2 different projects on 2 different machines exhibited the same behaviour. If you put localhost:4200 into another browser the project runs but it won't debug. Seems to be only the browser launched by vs code.
> 
> As far as I can tell this coincides with 1.76.1 being installed.
> 
> Is there a way to download older versions?

yes you can find old versions on https://code.visualstudio.com/updates/v1_76
Don't forget to change the auto update setting 😁let me known if it's work for you, I tried without success

**Comment by AndrewStaggPlunk** [comment]
Ok, i've got to root of the problem
Had a third machine that was running 1.75.1 and was working, but it using a slightly older version of Chrome: Upgraded VS Code to 1.76.1 and it still worked, So, then I upgraded Chrome to the latest version and all of a sudden the above behaviour occurred.

On all 3 machines Chrome is on 111,0.5563.65 and are now not working

**Comment by AndrewStaggPlunk** [comment]
Firefox loads up fine and breakpoints are hit
Chrome still not working, deciding on whether to downgrade Chrone

**Comment by connor4312** [comment]
Several of you are probably running into https://github.com/microsoft/vscode-js-debug/issues/1604

The others are probably hitting https://github.com/microsoft/vscode/issues/167353
